build.rs

```diff
@@ -159,8 +159,8 @@ fn write_testsuite_tests(
         writeln!(out, r#"#[should_panic]"#)?;
     } else if ignore(testsuite, &testname, strategy) {
         writeln!(out, "#[ignore]")?;
-    } else if pooling {
-        // Ignore on aarch64 due to using QEMU for running tests (limited memory)
+    } else if pooling && platform_is_emulated() {
+        // Ignore on aarch64 when using QEMU for running tests (limited memory)
         writeln!(out, r#"#[cfg_attr(target_arch = "aarch64", ignore)]"#)?;
```

Comment:
I just ran into this problem on s390x under QEMU as well.   Shouldn't we just unconditionally ignore the pooling tests on all platforms if platform_is_emulated() is true?